h2o-r/h2o-package/R/stackedensemble.R

```diff
@@ -34,6 +34,8 @@
 #'        classification problems. Must be one of: "AUTO", "Random", "Modulo", "Stratified".
 #' @param metalearner_fold_column Column with cross-validation fold index assignment per observation for cross-validation of the metalearner.
 #' @param metalearner_params Parameters for metalearner algorithm
+#' @param metalearner_transform Transformation used for the level one frame. Must be one of: "NONE", "Logit", "PercentileRank". Defaults to
```

Comment:
See below.

Reply:
@ledell thank you for mentioning the target encoding. I found there a way to change the values to snake case. When I wrote that comment, I assumed I would have to modify `h2o-bindings/bin/gen_R.py` which would change the behavior for all the generated enums. In target encoding, I found out that I can change just the one parameter.

But I found out that [1] checks it against CamelCase names from the backend in case insensitive manner so I modified it to get rid of underscores and then match which seems to be compatible with what we do in the backend[2].

I added a test in R to make sure it works also I tested it but it is last minute change so if it seems uncomfortably complicated, I am not against of removing the commit and using it later in a fix release.

[1] https://github.com/h2oai/h2o-3/blob/9ff5e74b185a59a6be5cdbe36959695b326088c1/h2o-r/h2o-package/R/models.R#L337

[2] https://github.com/h2oai/h2o-3/blob/18f3f0f522ef3f00c3f9983624f3280af5124d88/h2o-core/src/main/java/water/util/EnumUtils.java#L29-L39